chia/cmds/beta_funcs.py

```diff
@@ -1,3 +1,5 @@
+# Package: chia-blockchain
```

Comment:
I would have expected this to be part of the `cmds` package, but maybe the idea is to make all the click-handlers part of the tool they're used in, and the `cmds` package is just for shared command line utilities.

If `cmds` is meant to be shared tools for implementing click handlers, we would expect it to have no (or few) dependencies of its own, and only on low level basic stuff, like `Streamable`.

Reply:
That was my interpretation, correct.